metrics/firebase/test/firestore/rules/test_utils/test-data.js

```diff
@@ -50,11 +52,20 @@ const builds = {
   },
 };
 
+/** A list of test allowed email domains */
+const allowedEmailDomains = {
+  "allowed_email_domains/gmail.com": {},
+};
+
 exports.projectGroups = projectGroups;
 exports.builds = builds;
+exports.allowedEmailDomains = allowedEmailDomains;
 
 /** A firebase user needed for tests */
-exports.user = { uid: "uid" };
+exports.user = { uid: "uid", email: 'test@gmail.com' };
+
+/** A firebase user with an invalid email needed for tests */
+exports.invalidUser = { uid: "uid", email: 'test@invalid.com' };
```

Comment:
Should we have 2 users? 
1) sign_in_provider - email
2) sign_in_provider - other/Google